### DISCUSSION TITLE
JK BMS cable lenght (resistance)

### DISCUSSION THREAD

**Discussion by Dr-Gigavolt** [mr-manuel/venus-os_dbus-serialbattery#121] [General]
Dear users of JK BMS,
before I made my first experience with the JK BMS i didn't know how the cell balancing exactly works. When I built my LTO batteries 
[http://gigavolt.eu/projects/PV2/Batteries.jpg](url) 
I adjusted the resistance of measurement cables as equal as possible. The question is if it is necessary. Does the BMS measure the voltage of the two cells just exchanging charge? If yes, are the on-start measured resistances considered? I'm going to extend 22 to 24 cells in series (due to lack of experience at the beginning I over-estimated the cell voltage in full state and expected that S22 is the maximum for Victron). After this modification two of six Lapp cables will be too short, so I would need to change all six in order to maintain equal resistances. I don't want to exert this effort again if not necessary. Thanks a lot.
Best regards,
Anton

**Comment by Albarge** [comment]
I don't _think_ it should matter, but I'm not an electrical engineer, as long as they have a similar resistance to each other, the app shows their resistance, and as I think you found they can be calibrated,  but I'm not sure exactly how that affects it. I cut all my cables different lengths and still had good low resistance and similar values, so left it alone, but they're all generally short. I haven't noticed any threads on DIY solar where the resistances caused an obvious issue for the BMS calculations, or if there is an upper limit, but maybe it's possible.

Sorry don't see an option to make the pictures smaller in github

![Screenshot_20231207-184528](https://github.com/user-attachments/assets/0e491c35-0923-4fc8-be04-1d391469f8d9)

![Screenshot_20231207-184737](https://github.com/user-attachments/assets/295ed93c-2c03-44fe-8d0e-9f52a0eecd59)

![IMG_20231205_191529_562](https://github.com/user-attachments/assets/e91687f3-72db-41c1-a7b3-ec2d0c24dcfe)
